frontend/src/contexts/HostContext/HostReducer.spec.js

```diff
@@ -1,7 +1,5 @@
 import HostReducer from './HostReducer';
```

Comment:
I would probably add a test that checks that the function here works as expected.

Reply:
@wise-king-sullyman the import statement?